python/GafferUI/NodeSetEditor.py

```diff
@@ -61,22 +77,149 @@ def __init__( self, topLevelWidget, scriptNode, **kw ) :
 		# to avoid being called when they're only half constructed.
 		self.__setNodeSetInternal( self.scriptNode().selection(), callUpdateFromSet=False )
 
-	## Sets the nodes that will be displayed by this editor. As members are
-	# added to and removed from the set, the UI will be updated automatically
-	# to show them. This also calls `nodeSet.setRemoveOrphans( True )` so that
-	# deleted nodes are not visible in the UI.
+	## Sets the nodes that will be displayed by this editor.
```

Comment:
Restore original formatting.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/3270/commits/0f4055a0353ab13b52dd2c1589ce6c6d069d85a1